Sources/ComposableArchitecture/TestStore.swift

```diff
@@ -2316,6 +2316,20 @@ class TestReducer<State, Action>: ReducerProtocol {
         )
         .map { .init(origin: .receive($0), file: action.file, line: action.line) }
         .eraseToEffect()
+
+    case .run(let priority, let perform):
```

Comment:
we can't piggyback off of Combine operators here anymore because we lose the `SendTask` return value; conceptually speaking conversion to a `Publisher` is now lossy.